contxt/services/base_graph_service.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from sgqlc.endpoint.http import HTTPEndpoint
+from sgqlc.operation import Operation
+
+from contxt.auth import Auth
+from contxt.services.api import ApiEnvironment, ConfiguredApi
+
+
+class BaseGraphService(ConfiguredApi):
+
+    _envs = (
+        ApiEnvironment(
+            name="production",
+            base_url="https://ndustrial.api.ndustrial.io/graphql",
+            client_id="vtiZlMRo4apDvThTRiH7kLifQXWUdi9j",
+        ),
+        ApiEnvironment(
+            name="staging",
+            base_url="https://ndustrial.api.staging.ndustrial.io/graphql",
+            client_id="vhGxildn8hRRWZj49y18BGtbjTkFHcTG",
+        ),
+    )
+
+    org_id_slugs = {
```

Comment:
I'm not sure, I think a better long-term plan would be to have some kind of service-discovery in place. As for this, I was just going with what we did in the js sdk already

Reply:
In reality, we should be moving away from `orgId` in my mind. It will be a stale concept once all services are single tenant. Instead, you would access everything my tenant slug. This is fine for now, we will just have to update later one. Could we potentially allow passing either orgId or slug and just take the slug as-is?